src/stories/SimpleAutocomplete.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { action } from '@storybook/addon-actions';
+import { Story } from '@storybook/react';
+import React from 'react';
+import  {
+  SimpleAutocompleteProps,
+  SimpleAutocomplete,
+} from '../components/Autocomplete';
+
+export default {
+  title: 'SimpleAutocomplete',
+  component: SimpleAutocomplete,
+};
+
+const SimpleAutocompleteTemplate: Story<SimpleAutocompleteProps> = (args) => {
+  const [selected, setSelected] = React.useState('');
+  const handleChange = (id: string, value: string) => {
+    action('onChange')(value);
+    setSelected(value);
+  };
+
+  return <SimpleAutocomplete {...args} selected={selected} onChange={handleChange} />;
+};
+
+export const SimpleAutocompleteStory = SimpleAutocompleteTemplate.bind({});
```

Comment:
Can we change this 'SimpleAutocompleteStory' to something like 'Default'? This is the name that shows under 'SimpleAutocomplete'

Reply:
will do, i had the 2 autocomplete widgets in the same story and renamed them. forgot to switch back to Default when i split the stories.

Reply:
Thanks!